code/obj/machinery/computer/cloning.dm

```diff
@@ -183,14 +187,22 @@ ADMIN_INTERACT_PROCS(/obj/machinery/computer/cloning, proc/scan_someone, proc/cl
 		qdel(W)
 
 	else if (istype(W, /obj/item/cloneModule/minderaser))
-		if(allow_mind_erasure || allow_dead_scanning)
+		if(allow_mind_erasure || allow_dead_scanning || microbomb_upgrade)
```

Comment:
yeah this is pretty cringe, this should be refactored to just be handled on the module side